pulsar-broker/src/main/java/org/apache/pulsar/broker/TransactionMetadataStoreService.java

```diff
@@ -98,6 +107,7 @@ public TransactionMetadataStoreService(TransactionMetadataStoreProvider transact
         this.transactionOpRetryTimer = timer;
         this.tcLoadSemaphores = new ConcurrentLongHashMap<>();
         this.pendingConnectRequests = new ConcurrentLongHashMap<>();
+        this.internalPinnedExecutor = Executors.newSingleThreadScheduledExecutor(threadFactory);
```

Comment:
please shutdown this threadpool

Reply:
Thanks for pointing it out, it allowed me to correct this mistake.